Kitodo/src/test/java/org/kitodo/production/services/catalogimport/MassImportTest.java

```diff
@@ -30,11 +30,17 @@ public class MassImportTest {
     private static final String ID = "ID";
     private static final String TITLE = "Title";
     private static final String PLACE = "Place";
+    private static final String SECOND_PLACE = "Place";
```

Comment:
Why is this constant required if it only holds the same value as the previous constant `PLACE`? Since we explicitly want to test multiple values for the same metadata here, re-using the same constant for the additional values would make sense, too, wouldn't it?